lib/streamaggr/streamaggr.go

```diff
@@ -583,6 +584,7 @@ func newAggregator(cfg *Config, pushFunc PushFunc, ms *metrics.Set, opts *Option
 
 		flushTimeouts:      ms.GetOrCreateCounter(`vm_streamaggr_flush_timeouts_total`),
 		dedupFlushTimeouts: ms.GetOrCreateCounter(`vm_streamaggr_dedup_flush_timeouts_total`),
+		ignoredSamples:     ms.GetOrCreateCounter(`vm_streamaggr_ignored_samples_total`),
```

Comment:
I'd suggest introducing `reason` label to this metric. In this case, the reason to ignore samples is `too old timestamp`.
I think we need `reason` label, so later we can add `reason=NaN value` or `reason=dropped by relabeling`.